packages/cdk/lib/constants.ts

```diff
@@ -15,6 +15,7 @@ export const VPC_SUBNETS_PARAMETER_NAME = "InfraSubnets";
 export const VPC_NUMBER_SUBNETS_PARAMETER_NAME = "NumInfraSubnets";
 export const WES_KEY_PARAMETER_NAME = "WesAdapterZipKeyParameter";
 export const WES_BUCKET_NAME = "WesAdapterZipBucket";
+export const IMAGE_PARAMETER_NAME = "ComputeEnvImage";
```

Comment:
Should we call it "COMPUTE_IMAGE_PARAMETER_NAME" ?

Reply:
Yes, I can refactor that.